addons/website_event/controllers/main.py

```diff
@@ -280,15 +280,15 @@ def _process_attendees_form(self, event, form_details):
                 registration_index, field_name = key_values
                 if field_name not in registration_fields:
                     continue
-                registrations.setdefault(registration_index, dict())[field_name] = int(value)
+                registrations.setdefault(registration_index, dict())[field_name] = int(value) or False
                 continue
 
             registration_index, question_type, question_id = key_values
             answer_values = None
             if question_type == 'simple_choice':
                 answer_values = {
                     'question_id': int(question_id),
-                    'value_answer_id': int(value)
+                    'value_answer_id': int(value) or False
```

Comment:
Is this necessary? If so I don't think your test case covers it.

Reply:
Mm I guess there's no good reason it should be 0 so it would be best to fail early either way. Will revert.